test/functional/url_parser_tests.js

```diff
@@ -426,7 +426,27 @@ exports['Parse mongodb://localhost/db?w=-1'] = {
 /**
  * @ignore
  */
-exports['Throw on unsuported options'] = {
+exports['Should be able to parse mongodb://localhost/?compressors=snappy,zlib&zlibCompressionLevel=-1'] = {
```

Comment:
nitpick: I think we can name this test better. I know you've used the same form used elsewhere in the document here, but I think it's better to clearly indicate what part of uri parsing you're actually testing here.